app/konnect/getting-started/add-api.md

```diff
@@ -0,0 +1,160 @@
+
+1. Get the [list of all control planes](/konnect/api/control-planes/latest/#/Control%20Planes/list-control-planes) so you can copy the control plane ID associated with the `default` control plane:
+  ```sh
+  curl --request GET \
+    --url https://us.api.konghq.com/v2/control-planes \
+    --header 'Authorization: Bearer <personal-access-token>' \
+    --header 'accept: application/json'
+  ```
+  In this guide, we will assign your service to the `default` control plane for the sake of simplicity.
+
+1. [Create a service](/konnect/api/control-plane-configuration/latest/#/Services/create-service) for your API that is assigned to the `default` control plane:
+  ```bash
+  curl --request POST \
+    --url https://{region}.api.konghq.com/v2/control-planes/{defaultControlPlaneId}/core-entities/services \
+    --header 'Authorization: Bearer <personal-access-token>' \
+    --header 'Content-Type: application/json' \
+    --header 'accept: application/json' \
+    --data '{
+        "name": "my_service",
+        "host": "httpbin.org",
+        "path": "/my_api"
+    }'
+  ```
+  This service represents your backend API. Be sure to replace the PAT as well as the following placeholders with your own values:
+  * `defaultControlPlaneId`: The ID of the default control plane. This associates the service with that control plane.
+  * `name`: The name you want to display for your service.
+  * `host`: The host of the upstream server. This is case sensitive.
+  * `path`: The path to be used in requests to the upstream server.
+  Be sure to save the service ID from the response to use it in the next step.
+
+1. [Add a route](/konnect/api/control-plane-configuration/latest/#/Routes/create-route) to your service:
+  ```bash
+  curl --request POST \
+    --url https://{region}.api.konghq.com/v2/control-planes/{defaultControlPlaneId}/core-entities/routes \
+    --header 'Authorization: Bearer <personal-access-token>' \
+    --header 'Content-Type: application/json' \
+    --header 'accept: application/json' \
+    --data '{
+        "name": "my_route",
+        "hosts": [
+          "httpbin.org"
+        ],
+        "paths": [
+          "/my_api"
+        ],
+        "service": {
+          "id": "49248arf-b90c-4c0b-9529-3a949dfc10d1"
+        }
+    }'
+  ```
+  This route defines what is exposed to clients. Be sure to replace the PAT as well as the following placeholders with your own values:
+  * `defaultControlPlaneId`: The ID of the default control plane. This associates the service with that control plane.
+  * `name`: The name you want to display for your route.
+  * `host`: A list of domain names that match this route. This is case sensitive.
+  * `path`: A list of paths that match this route.
+  * `"service": "id"`: The ID of the service you created in the previous step. This should be part of the response from the previous request.
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Verify the implementation
```

Comment:
I left this in from the original doc, but it wasn't quite working for me. Is it because this still contains httpbin/mock?

Reply:
No, this should work if everything was set up correctly. [httpbin](https://httpbin.org/) is an active service, and `mock` is an arbitrary name for the route. 

Looks like the path was set to `/my_api` and not `/mock` in the previous steps? So the test should be `http://localhost:8000/my-api`.